src/simulators/qasm/qasm_controller.hpp

```diff
@@ -394,7 +414,7 @@ void QasmController::initialize_state(const Circuit &circ,
   if (initial_state.empty()) {
     state.initialize_qreg(circ.num_qubits);
   } else {
-    state.initialize_qreg(circ.num_qubits, initial_state);
+    //state.initialize_qreg(circ.num_qubits, initial_state);
```

Comment:
What's the reason for removing this line?

Reply:
I temporarily commented this out because of a problem I had. Now I put it back in.

Reply:
I put back this line. It was a temporary removal.